examples/tensorboard_simple.py

```diff
@@ -0,0 +1,62 @@
+import optuna
+from optuna.integration.tensorboard import TensorBoardCallback
+from optuna import type_checking
+import tensorflow as tf
+
+if type_checking.TYPE_CHECKING:
+    from typing import Dict  # NOQA
+    from typing import Any  # NOQA
+
+
+fashion_mnist = tf.keras.datasets.fashion_mnist
+
+(x_train, y_train), (x_test, y_test) = fashion_mnist.load_data()
+x_train, x_test = x_train / 255.0, x_test / 255.0
+
+
+def train_test_model(num_units, dropout_rate, optimizer):
+    # type: (int, float, str) -> float
```

Comment:
We can use the new style for the type hinting. For example, please refer to [here](https://github.com/optuna/optuna/blob/master/optuna/pruners/hyperband.py).